packages/server/modules/workspaces/utils/mapWorkspaceRoleToProjectRole.ts

```diff
@@ -0,0 +1,16 @@
+import { StreamRoles, WorkspaceRoles } from '@speckle/shared'
+
+/**
+ * Given a user's workspace role, return the role they should have for workspace projects.
+ */
+export const mapWorkspaceRoleToProjectRole = (
+  workspaceRole: WorkspaceRoles
+): StreamRoles => {
+  switch (workspaceRole) {
+    case 'workspace:guest':
```

Comment:
avoid string literals, use the `Roles` const from @speckle/shared

Reply:
roger, done